cibyl/sources/source_factory.py

```diff
@@ -42,10 +45,18 @@ class SourceFactory:
 
     @staticmethod
     def extend_source(source):
+        source_class = ""
         if source.__name__ == 'Jenkins':
+            source_class = Jenkins
+        elif source.__name__ == 'ElasticSearch':
+            source_class = ElasticSearch
+
+        if source_class:
```

Comment:
I think it would be interesting to log the case where the source class is unknown. If I come here with the Zuul source, for example, it is quietly skipping something that should probably happen.